extensions/amp-amazon-onetag/0.1/test/validator-amp-amazon-onetag.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
It's a bit silly, but please add a copyright statement to the test cases. You can see other extension tests for an example.